quiz-bundle-spec.md

```diff
@@ -28,6 +28,7 @@ default_locale     | ✓        | string     | Corresponds to the locale that th
 schema_version     | ✓        | integer    | Which version of the quiz bundle schema you are using
 title              |          | dictionary | A locale dictionary of the quiz title, such as "My Awesome Quiz"
 passing_percentage | ✓        | integer    | The threshold grade that a student needs to achieve in order to count as "passing" the quiz.
+duration           |          | integer    | The maximum time a student is allotted for the quiz, in minutes.
```

Comment:
```suggestion
duration           |          | integer    | The default time a student is allotted for the quiz, in minutes.
```

Reply:
Also we should make it clear that the default is for the quiz to be untimed.

Reply:
Done.